### PR TITLE
chore: update supported versions 

### PR DESCRIPTION
**Pull request by arkaitzgarro**
This drops support for Laravel 5.5 and adds support for Laravel 8.x. An alternative for #101 but using GH actions.

Circleci did work well until now, but the support for matrixes is far from ideal, making it hard to add and maintain new versions of PHP and Lavarel. The way GH manages is more cleaner and easier to manage.

### CODE REVIEW

**Comment by countless-integers** [comment]
Circle build is failing 😉

**Comment by arkaitzgarro** [comment]
> Circle build is failing 😉

Yes, I've just removed the integration. I'm afraid the error will stay unless I push a new commit. Let me see...

**Review by countless-integers — APPROVED** [review]
🎉 
:octocat:

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                restore-keys: |
+                  ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}-${{ matrix.php }}-php${{ matrix.php }}
+                  ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}-${{ matrix.php }}
+                  ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
+                  ${{ runner.os }}-composer-
+
+            - name: Install dependencies
+              run: |
+                  composer require "laravel/framework:${{ matrix.laravel }}" "orchestra/testbench:${{ matrix.testbench }}" --no-interaction --no-update
+                  composer update --${{ matrix.dependency-version }} --prefer-dist --no-interaction --no-suggest
```
Comment (dstepe) [inline .github/workflows/run-tests.yml]: I think you want to remove `--${{ matrix.dependency-version }}` from the update command. That parameter is not being used and causes strange behaviors in the job, though not failures:

```
Run composer require "laravel/framework:8.*" "orchestra/testbench:6.*" --no-interaction --no-update
  composer require "laravel/framework:8.*" "orchestra/testbench:6.*" --no-interaction --no-update
  composer update -- --prefer-dist --no-interaction --no-suggest
  shell: /bin/bash -e {0}
./composer.json has been updated
Package "--prefer-dist" listed for update is not installed. Ignoring.
Package "--no-interaction" listed for update is not installed. Ignoring.
Package "--no-suggest" listed for update is not installed. Ignoring.
```

Comment (arkaitzgarro) [reply .github/workflows/run-tests.yml]: True, this is a leftover from previous experiments 👍

**Pull request merged**